config/paths.json

```diff
@@ -8,5 +8,19 @@
   "dist": "dist/",
   "packages": "packages/",
   "public": "public/",
-  "src": "src/"
+  "src": "src/",
+  "entry": {
```

Comment:
Could we link to `src/all/all.js` from here and explain if user should add/remove components javascripts in this file or in `src/all/all.js`?